langchain4j-core/src/test/java/dev/langchain4j/model/LambdaStreamingResponseHandlerTest.java

```diff
@@ -5,12 +5,14 @@
 import dev.langchain4j.model.chat.response.StreamingChatResponseHandler;
 import org.assertj.core.api.WithAssertions;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.Timeout;
 
 import java.util.ArrayList;
 import java.util.List;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.atomic.AtomicBoolean;
 
-import static dev.langchain4j.model.LambdaStreamingResponseHandler.onPartialResponse;
-import static dev.langchain4j.model.LambdaStreamingResponseHandler.onPartialResponseAndError;
+import static dev.langchain4j.model.LambdaStreamingResponseHandler.*;
```

Comment:
We tend to not use wildcards in the codebase, so individual static imports would be better.

Reply:
I just replaced the * with explicit methods for the static import.